### PR TITLE
feat(data-table): Add DataTableManager

### PR DESCRIPTION
**Pull request by jonnybel**
#### Summary

This is basically the same PR as https://github.com/commercetools/ui-kit/pull/1416 by @NickDevG, but pointing to the main branch and running the VRT tests.

### CODE REVIEW

**Comment by jonnybel** [comment]
We should add some VRTs. I'm on it.

**Comment by emmenko** [comment]
Ok, maybe the follow ups are 1. add additional VRTs and 2. refactor the tests.

So that refactoring the tests is backed up by the VRTs.

**Comment by emmenko** [comment]
@jonnybel feel free to merge this whenever you're ready.

File: packages/components/data-table-manager/src/data-table.visualroute.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  {
+    id: '1-parasite',
+    title: 'Parasite',
+    year: 2019,
+    director: 'Bong',
+    country: 'South Korea',
+  },
+  {
+    id: '2-woman',
+    title: 'Woman At War',
```
Comment (tdeekens) [inline packages/components/data-table-manager/src/data-table.visualroute.js]: I watched the other two. This one still not. I should listen more to you.

**Pull request merged**